Tests/SwiftCurrent_SwiftUITests/SwiftCurrent_SwiftUITests.swift

```diff
@@ -999,3 +992,20 @@ protocol StateIdentifiable { }
 extension State: StateIdentifiable {
 
 }
+
+fileprivate extension XCTestCase {
+    func decodeAnyWorkflow(with sequence: WorkflowDecodable.Type...) throws -> AnyWorkflow {
+        try JSONDecoder().decodeWorkflow(withAggregator: TestRegistry(types: sequence), from: generateValidWorkflowSpecification(with: sequence))
+    }
+
+    func generateValidWorkflowSpecification(with sequence: [WorkflowDecodable.Type]) throws -> Data {
```

Comment:
I wonder if this should be extracted further out for reuse elsewhere?

Reply:
The idea is that workflows will be `Encodable` in the near future, making this entire method unnecessary